Add integration tests for elasticsearch

The regression #2241 made me think that we should implement some IT for elasticsearch REST endpoints

I created PR #3168 and tests are ok, but some jobs are failed (Cannot find module 'phantomjs-prebuilt') ; seems like that's nothing to do with my changes -> in my opinion the PR is ok

--- a/generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java
+++ b/generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java
@@ -378,16 +378,15 @@ public class <%= entityClass %>ResourceIntTest <% if (databaseType == 'cassandra
         // Get the <%= entityInstance %>
         rest<%= entityClass %>MockMvc.perform(delete("/api/<%= entityApiUrl %>/{id}", <%= entityInstance %>.getId())
                 .accept(TestUtil.APPLICATION_JSON_UTF8))
-                .andExpect(status().isOk());
+                .andExpect(status().isOk());<% if (searchEngine == 'elasticsearch') { %>
+
+        // Validate ElasticSearch is empty
+        boolean <%= entityInstance %>ExistsInEs = <%= entityInstance %>SearchRepository.exists(<%= entityInstance %>.getId());
+        assertThat(<%= entityInstance %>ExistsInEs).isFalse();<% } %>
 
         // Validate the database is empty
         List<<%= entityClass %>> <%= entityInstancePlural %> = <%= entityInstance %>Repository.findAll();
-        assertThat(<%= entityInstancePlural %>).hasSize(databaseSizeBeforeDelete - 1);<% if (searchEngine == 'elasticsearch') { %>
-
-        // Validate tElasticSearch is empty
-        boolean <%= entityInstance %>ExistsInEs = <%= entityInstance %>SearchRepository.exists(<%= entityInstance %>.getId());
-        assertThat(<%= entityInstance %>ExistsInEs).isFalse();
-        <%_ } _%>
+        assertThat(<%= entityInstancePlural %>).hasSize(databaseSizeBeforeDelete - 1);
     }<% if (searchEngine == 'elasticsearch') { %>
 
     @Test<% if (databaseType == 'sql') { %>

